NewTutorials/Tutorial/SetupTutorials.swift

```diff
@@ -0,0 +1,32 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+func setupPlayingVideos(vc: TutorialCasesViewController) {
+    func process(player: Future<Result<Player>>) {
+        guard let wrapper = vc.storyboard?.instantiateViewController(withIdentifier: "PlayerViewControllerWrapper") as? PlayerViewControllerWrapper else {
+            fatalError("Wrapper not found")
+        }
+        wrapper.props.isLoading = true
```

Comment:
Could you just pass a player future to the wrapper and control loading process there? It is internal stuff.